### PR TITLE
Moving FAQ  Purging dataset

### PR DESCRIPTION
**Pull request by Nurzhamalyrys**
Hi! 
I am moving FAQ Purging dataset

### CODE REVIEW

**Review by gallardoalba — APPROVED** [review]

**Comment by gallardoalba** [comment]
Hi @Nurzhamalyrys! Thanks for the contributions. Could you add yourself to the contributors file? https://github.com/galaxyproject/training-material/blob/main/CONTRIBUTORS.yaml

**Comment by Nurzhamalyrys** [comment]
> Hi @Nurzhamalyrys! Thanks for the contributions. Could you add yourself to the contributors file? https://github.com/galaxyproject/training-material/blob/main/CONTRIBUTORS.yaml

Hi! I want to inform you that I added myself as a contributor in another pull request #3359

File: faqs/galaxy/datasets_purgin_datasets.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: purging datasets
+area: datasets     
+box_type: tip        
+layout: faq        
+contributors: [nurzhamalyrys] 
+---
+
+1. All account Datasets can be reviewed under User > Datasets.
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_purgin_datasets.md:9]:
```suggestion
1. All account Datasets can be reviewed under **User > Datasets**.
```
```suggestion
1. All account Datasets can be reviewed under User > Datasets.
```

Comment (beatrizserrano) [inline faqs/galaxy/datasets_purgin_datasets.md:9]:
```suggestion
1. All account Datasets can be reviewed under **User > Datasets**.
```
```suggestion
1. All account Datasets can be reviewed under User > Datasets.
```

File: faqs/galaxy/datasets_purgin_datasets.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+2. To permenently delete: use the link from within the dataset, or use the Operations on Multiple Datasets functions, or use the Purge Deleted Datasets option in the History menu.
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_purgin_datasets.md:10]:
```suggestion
2. To permanently delete: use the link from within the dataset, or use the Operations on Multiple Datasets functions, or use the Purge Deleted Datasets option in the History menu.
```
```suggestion
2. To permenently delete: use the link from within the dataset, or use the Operations on Multiple Datasets functions, or use the Purge Deleted Datasets option in the History menu.
```

File: faqs/galaxy/datasets_purgin_datasets.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Notes:
+- Within a History, deleted/permenently deleted Datasets can be reviewed by toggling the deleted link at the top of the History panel, found immediately under the History name.
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_purgin_datasets.md:13]:
```suggestion
- Within a History, deleted/permanently deleted Datasets can be reviewed by toggling the deleted link at the top of the History panel, found immediately under the History name.
```
```suggestion
- Within a History, deleted/permenently deleted Datasets can be reviewed by toggling the deleted link at the top of the History panel, found immediately under the History name.
```

Comment (beatrizserrano) [inline faqs/galaxy/datasets_purgin_datasets.md:13]:
```suggestion
- Within a History, deleted/permanently deleted Datasets can be reviewed by toggling the deleted link at the top of the History panel, found immediately under the History name.
```
```suggestion
- Within a History, deleted/permenently deleted Datasets can be reviewed by toggling the deleted link at the top of the History panel, found immediately under the History name.
```

File: faqs/galaxy/datasets_purgin_datasets.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+- Both active (shown by default) and hidden (the other toggle link, next to the deleted link) datasets can be reviewed the same way.
+- Click on the far right "X" to delete a dataset.
+- Datasets in a deleted state are still part of your quota usage.
+- Datasets must be purged (permenently deleted) to not count toward quota.
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_purgin_datasets.md:17]:
```suggestion
- Datasets must be purged (permanently deleted) to not count toward quota.
```

**Review by annefou — DISMISSED** [review]

**Review by beatrizserrano — DISMISSED** [review]

**Review by annefou — APPROVED** [review]

**Comment by beatrizserrano** [comment]
thank you @Nurzhamalyrys!!

**Pull request merged**